projects/plugins/jetpack/changelog/update-move-wpcom-v2-publicize-connections-to-publicize

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
Do you think this should be more significant for this kind of change, or not because it's backwards compatible?

Reply:
Yes, a patch because it's backwards compatible.